packages/vinxi-mdx/package.json

```diff
@@ -9,19 +9,26 @@
     "// TypeScript": "",
     "tsc:watch": "tsc --build --watch",
     "tsc:once": "tsc --build",
-    "build:cjs": "esbuild --bundle --platform=node --external:esbuild --external:vite --external:@mdx-js/mdx src/index.ts > dist/index.cjs",
+    "build:cjs": "esbuild --bundle --platform=node --external:esbuild --external:vite --external:@mdx-js/mdx --outfile=dist/index.cjs --sourcemap=linked dist/index.js",
     "// Release": "",
-    "prepublishOnly": "pnpm run build:cjs",
+    "prepublishOnly": "pnpm run types && pnpm run build:cjs && node ./scripts/update-exports-prepublish.js",
+    "postpublish": "node ./scripts/update-exports-postpublish.js",
     "release": "npm version patch && npm publish && git push && git push --tags",
     "// Remove caches": "",
     "clean": "git clean -Xdff"
   },
   "main": "dist/index.cjs",
+  "exports": {
+    ".": {
+      "require": "./src/index.ts",
+      "import": "./src/index.ts"
+    }
```

Comment:
By not including this esm was never being utilized in environments that support it.